features/support/env.rb

```diff
@@ -1,11 +1,58 @@
+# IMPORTANT: This file is generated by cucumber-rails - edit at your own peril.
```

Comment:
Why are we adding all the comments on this file again? 🤔

Reply:
Done